examples/tutorials/nb_python/ECCV_2020_Navigation.py

```diff
@@ -0,0 +1,1094 @@
+def convert_points_to_topdown(pathfinder, points, meters_per_pixel):
+    points_topdown = []
+    bounds = pathfinder.get_bounds()
+    for point in points:
+        # convert 3D x,z to topdown x,y
+        px = (point[0] - bounds[0][0]) / meters_per_pixel
+        py = (point[2] - bounds[0][2]) / meters_per_pixel
+        points_topdown.append(np.array([px, py]))
+    return points_topdown
+
+
+def display_topdown_map(topdown_map, trajectory=None, key_points=None):
+    size = topdown_map.shape[::-1]
+    databytes = np.packbits(topdown_map, axis=1)
+    databytes = databytes.copy(order="C")
+    topdown_img = Image.frombytes(mode="1", size=size, data=databytes)
+    plt.figure(figsize=(12, 8))
+    ax = plt.subplot(1, 1, 1)
+    ax.axis("off")
+    # ax.set_title('topdown map')
+    # plot a trajectory (image space) on the map
+    if trajectory is not None:
+        if len(trajectory) > 1:
+            # plot the trajectory
+            ax.plot(
+                [p[0] for p in trajectory],
+                [p[1] for p in trajectory],
+                linewidth=2,
+                color="firebrick",
+            )
+            # plot the agent icon
+            tangent = unit_vector(trajectory[1] - trajectory[0])
+            initial_angle = math.atan2(tangent[1], tangent[0]) * 180 / math.pi
+            plt.plot(
+                trajectory[0][0],
+                trajectory[0][1],
+                marker=(3, 0, -initial_angle - 90),
+                markersize=20,
+                linestyle="None",
+                alpha=0.8,
+            )
+    # plot points on map
+    if key_points is not None:
+        for pix, point in enumerate(key_points):
+            plt.plot(point[0], point[1], marker="o", markersize=10, alpha=0.8)
+    # draw axis
+    x = [1, 10]
+    y = [1, 1]
+    # ax.plot(x,y, linewidth=2, color='red')
+    # ax.plot(y,x, linewidth=2, color='green')
+
+    plt.imshow(topdown_img)
+    plt.show()
+
+
+try:
+    # @markdown ###Configure Example Parameters:
+    # @markdown Configure the map resolution:
+    meters_per_pixel = 0.1  # @param {type:"slider", min:0.01, max:1.0, step:0.01}
+    # @markdown ---
+    # @markdown Customize the map slice height (global y coordinate):
+    custom_height = False  # @param {type:"boolean"}
+    height = 2.8  # @param {type:"slider", min:-10, max:10, step:0.1}
+    # @markdown If not using custom height, default to scene lower limit.
+    # @markdown (Cell output provides scene height range from bounding box for reference.)
+
+    # get scene bounding box for specified height slice
+    scene_bb = sim.get_active_scene_graph().get_root_node().cumulative_bb
+    print("The Scene asset height range is: " + str(scene_bb.y()))
+
+    if not custom_height:
+        height = scene_bb.y().min
+
+    # This map is a 2D boolean array
+    topdown_map = sim.pathfinder.get_topdown_view(meters_per_pixel, height)
+    display_topdown_map(topdown_map)
+
+except:
+    print("Error - aborting")
```

Comment:
You may want to print the error traceback here using the traceback module. That way the user can debug